release-team/role-handbooks/branch-manager/README.md

```diff
@@ -115,6 +116,42 @@ As the first build from the new release branch, publication this build does not
 Adding the --rc flag switches behavior on to building release candidates.  Again the gcbmgr/anago scripting automatically finds and increments the current build number.
 * `./gcbmgr stage release-1.12 --rc --build-at-head --nomock`
 
+In an perfect world, `rc.1` and the official release are the same commit. To get as close to that as we can the following thing should be considered:
+
+1. All the PRs in the milestone should merge to master
+
+   [`is:pr is:open milestone:v1.14`][pr-milestone-query] should be empty.
```

Comment:
Please correct me if I'm wrong, but does this step mean that I should go to the Kubernetes repo's PR tab and search for `is:pr is:open milestone:v1.14` `<optional pr title>`  ?

And when you mean "empty", you're referring to the search results returning none. That is, there should be no more open PRs with the specified milestone tag, so all these should be closed/merged by then.

Reply:
oh okay. I just saw the url on line 152.